src/J1/S/P0011/View/Validate.java

```diff
@@ -0,0 +1,69 @@
+package J1.S.P0011.View;
+
+import java.util.Scanner;
+
+public class Validate {
+
+    private static final Scanner sc = new Scanner(System.in);
+    private static final String BINARY_VALID = "[0-1]*";
+    private static final String HEXADECIMAL_VALID = "[0-9a-fA-F]*";
+    private static final String DECIMAL_VALID = "[0-9]*";
+
+    public int checkInputIntLimit (int min, int max) {
+       while (true) {
+           try {
+               int result = Integer.parseInt(sc.nextLine().trim());
+               if (result < min && result > max) {
+                   throw new NumberFormatException();
```

Comment:
Không rõ nghĩa lắm, min max liên quan gì đến numberformat.

Reply:
Nên sửa show log ra là                System.out.println("Input number must be between " + min + " and " + max ); sau đó continute luôn